tests/Controller/AdminResettingControllerTest.php

```diff
@@ -29,4 +29,54 @@ public function testItIsInstantiable(): void
     {
         $this->assertNotNull($this->controller);
     }
+
+    /**
+     * Tests if the mail template has been properly divided into subject and body.
+     *
+     * @dataProvider emailTemplateData
+     */
+    public function testSendEmailMessage($template, $subject, $body): void
+    {
+        $mailer = $this->createMock('Swift_Mailer');
+
+        $receivedMessage = null;
+        $mailer->expects($this->once())->method('send')->with($this->callback(
+            function ($value) use (&$receivedMessage) {
+                $receivedMessage = $value;
+
+                return true;
+            }
+        ));
+
+        $controller = $this->getMockBuilder(AdminResettingController::class)
+            ->setMethods(['get'])
+            ->getMock();
+        $controller->expects($this->once())->method('get')->with('mailer')
+            ->will($this->returnValue($mailer));
```

Comment:
You could use https://github.com/sebastianbergmann/phpunit/blob/64d377cc488cd3707f34f46266c8d22e21f1da06/src/Framework/TestCase.php#L1323 here